testing/fuchsia/test_fars

```diff
@@ -1,12 +1,11 @@
-flutter_runner_tests-0.far
-flutter_runner_scenic_tests-0.far
-flutter_runner_tzdata_tests-0.far
+testing_tests-0.far
```

Comment:
I re-ordered the tests in this file deliberately.

This way they run in order of complexity, with simpler/fewer-dependencies tests running first.  the last test is the most complex conceptually and has the most dependencies.

Reply:
Nevermind, reverted this